### DISCUSSION TITLE
Account has been limited & marked as "Spammy"; No response from support team

### DISCUSSION THREAD

**Discussion by CarrotManMatt** [community/community#135406] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

On July 29th I noticed that my commits were being marked as unverified despite me using a correct GPG key to sign my commits. I was very confused as I had ensured that my email address in the GPG key matched the email account associated with my GitHub account. Only after viewing the reason for the unverified commits did I realise that it stated "No account matches this email address". Well after a lot of searching, and attempting to view my account page on an incognito account I saw that my account was not available to be viewed by other users.

So it seems that my account has been "limited". I can still log in and view all my own contributions (personal repos, comments, issues, PRs, etc.) but noone ales can view them. I have had no contact from GitHub about this happening, not the reason for it. The only place I can possibly gleam any information is that on my PR approval comments there is a small tag attached to the comment saying "Spammy".

I submitted a support ticket on the evening of July 29th to reinstate my account and ask any details on why this happened. Had I broken the terms of service? I have no idea what could have caused this.

It has been well over 7 working days since I submitted this support ticket (the possible amount of time it is suggested that it may take the support team to respond), yet I have had no response. I am appealed at the lack of communication, both as soon as the actions was taken on my account & not responding to my reinstatement request. It has severely affected my work and ability to contribute to projects on GitHub. (And before anyone suggests, no I can't move to another remote code hoster, as the projects I contribute to are based on GitHub and will not move just for me). What can I do to get my account reinstated?

TL;DR: My account was limited and marked as spammy on July 29th when I created a support ticket. I have had no response about what has happened. What can I do to get my account reinstated?
